src/Framework/Framework/Compilation/ControlTree/IAbstractDirectiveAttributeReference.cs

```diff
@@ -0,0 +1,14 @@
+﻿#nullable enable
+
+using DotVVM.Framework.Compilation.Parser.Binding.Parser;
+
+namespace DotVVM.Framework.Compilation.ControlTree
+{
+    public interface IAbstractDirectiveAttributeReference
+    {
+        public TypeReferenceBindingParserNode TypeSyntax { get; }
+        public IdentifierNameBindingParserNode NameSyntax { get; }
+        public LiteralExpressionBindingParserNode Initializer { get; }
+        public ITypeDescriptor? Type { get; }
```

Comment:
```suggestion
        TypeReferenceBindingParserNode TypeSyntax { get; }
        IdentifierNameBindingParserNode NameSyntax { get; }
        LiteralExpressionBindingParserNode Initializer { get; }
        ITypeDescriptor? Type { get; }
```

Reply:
Maybe add a short comment explaining what this node represents. It wasn't clear to me immediately :sweat_smile:

Reply:
Added comments removed public